plugin.program.remote.control.browser/resources/language/English/strings.po

```diff
@@ -0,0 +1,132 @@
+# XBMC Media Center language file
```

Comment:
would be cool if you could use the new folder naming scheme for language folders (in this case resource.language.en_gb )